coralproject/talk
# [CORL-1504] prevent selection of irrelevant options in slack config

## What does this PR do?

disables and un-checks other options for slack triggers when the "all comments" trigger is checked



## What changes to the GraphQL/Database Schema does this PR introduce?
none


## How do I test this PR?

check one or more checkboxes for slack triggers, then select "all comments". other checkboxes should be unchecked and disabled.
